### ISSUE TITLE
Router logits

### ISSUE THREAD

**Issue by ZhaozwTD** [zai-org/GLM-V#191]
请问 基于hf推理或者vllm推理过程中，有什么办法能拿到 router 对各个 expert 的 logits 嘛

**Comment by zRzRzRzRzRzRzR** [comment]
可以在这里打印，但是目前是没有返回（写法和DS一样）

<img width="1952" height="1086" alt="Image" src="https://github.com/user-attachments/assets/6bd66bb7-31b1-4a24-a92d-42727a4dbf8c" />
